### ISSUE TITLE
Editor settings

### ISSUE THREAD

**Issue by lpequignot** [newcat/baklavajs#280]
Hi, 

I'm testing version 2. and I'm not able to display the minimap with the following code.

``` 
const baklava = useBaklava();
baklava.settings.enableMinimap = true;
const editor = baklava.editor;
(window as any).editor = baklava.editor;
```

Also, I'm wondering if it would be possible to hide the toolbar and the sidebar ?

Thank you,

Louise

**Comment by newcat** [comment]
Minimap is currently disabled unfortunately :( for some reason the watchers needed to update the minimap drop the performance significantly. Until I found a solution for this, I decided to disable the minimap. But it is on the roadmap and should be working again for the final release of version 2.

Regarding toolbar+sidebar: It should be possible to hide both by passing either an empty template or `<div>` inside their respective slots: https://v2.baklava.tech/visual-editor/customization.html#custom-components
